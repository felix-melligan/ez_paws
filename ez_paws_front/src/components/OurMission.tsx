import React, { Component } from 'react';

class OurMission extends Component {
    render() {
        return(
            <h1>
                Our Mission
            </h1>
        );
    }
}

export default OurMission;