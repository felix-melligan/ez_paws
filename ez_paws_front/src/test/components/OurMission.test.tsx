import React from 'react';
import { shallow } from 'enzyme';

import OurMission from '../../components/OurMission';

let wrapper;

beforeEach(() => {
  wrapper = shallow(<OurMission />);
})

describe('<OurMission />', () => {
  it('Renders correctly', () => {
    expect(wrapper).toMatchSnapshot();
  });

  describe('Renders correct content', () => {

  });
});
