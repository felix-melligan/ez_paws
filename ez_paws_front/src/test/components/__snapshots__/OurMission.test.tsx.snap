// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<OurMission /> Renders correctly 1`] = `ShallowWrapper {}`;
